src/check-compatibility.ts

```diff
@@ -0,0 +1,67 @@
+import { Connection, Client } from "@temporalio/client";
+import { Worker } from "@temporalio/worker";
+import {
+  getConnectionOptions,
+  getWorkflowOptions,
+  namespace,
+  taskQueue,
+} from "./env";
+
+async function run() {
+  const prTitle = process.argv[2];
+  if (!prTitle) {
+    throw new Error(`usage: ${process.argv[1]} <prTitle>`);
+  }
+  const matches = prTitle.match(/^\[compatible-([^\]]+)\]/);
```

Comment:
nit: You'd probably not want `[` too

Reply:
I think it's good enough, there might be other characters that we don't want but I'll leave it as an exercise for the reader.